### PR TITLE
Add propagation of the exit code and err to the main command

### PR DESCRIPTION
**Pull request by zoltrain**
Solves: The pat command always returning 0 even if the tests fail.

I'm not sure what you want to do with the error propagated from "run" but at the moment I'm just printing it out and then exited 0.

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -64,23 +64,26 @@ func run(testFileGlobs []string) error {
...
 }
 
 // RootCmd represents the base command when called without any subcommands
 var RootCmd = &cobra.Command{
 	Use:   "pat <test-file-globs...>",
 	Short: "Prometheus Alert Testing utility",
 	Run: func(cmd *cobra.Command, args []string) {
-		run(args)
+		if code, err := run(args); err != nil {
+			fmt.Printf("exited with %s", strconv.Itoa(code))
```
Comment (kevinjqiu) [inline cmd/root.go]: Why not use `%d` and get rid of `Itoa` call?

**Comment by kevinjqiu** [comment]
@zoltrain Thanks.  It's a good addition.  I added some (minor) comments.

File: cmd/root.go (hunk 1 above)
Comment (zoltrain) [reply cmd/root.go]: That's a pretty good idea :-)

Change pushed.

**Comment by kevinjqiu** [comment]
@zoltrain Thanks.  Merged.

**Pull request merged**